plugins/node/opentelemetry-instrumentation-koa/examples/src/client.ts

```diff
@@ -16,7 +20,9 @@ function makeRequest() {
       span.setStatus({ code: api.SpanStatusCode.OK });
       console.log(res.statusText);
     } catch (e) {
-      span.setStatus({ code: api.SpanStatusCode.ERROR, message: e.message });
+      if(e instanceof Error) {
```

Comment:
This new `if` means that the span status might not be set to `ERROR` when an exception is thrown. Is this intentional? if so, can you explain the motivation?

Reply:
I took this behavior from https://github.com/open-telemetry/opentelemetry-js-contrib/pull/939/files (see the changes in client.ts file)
without this, it doesn't compile because there is no type for the `e` variable and I can't access its `.message` property.

Reply:
ok. 
I still think this is not the right pattern, but we can address this in a separate PR 👍